
Currently Loaded Modules:
  1) tools/miniconda/python3.8/4.9.2

 

WARNING: overwriting environment variables set in the machine
overwriting variable nnUNet_raw_data_base nnUNet_preprocessed RESULTS_FOLDER OUTPUT
  Running command git clone -q https://github.com/FabianIsensee/hiddenlayer.git /tmp/pip-install-l1mew5ya/hiddenlayer_6be3f22489d44d5c80e78c3aa932d8d9
  Running command git checkout -b more_plotted_details --track origin/more_plotted_details
  Switched to a new branch 'more_plotted_details'
  Branch 'more_plotted_details' set up to track remote branch 'more_plotted_details' from 'origin'.
Traceback (most recent call last):
  File "/home/smaijer/.conda/envs/nn/bin/nnUNet_train", line 33, in <module>
    sys.exit(load_entry_point('nnunet', 'console_scripts', 'nnUNet_train')())
  File "/home/smaijer/nnUNet/nnunet/run/run_training.py", line 164, in main
    trainer.initialize(not validation_only)
  File "/home/smaijer/nnUNet/nnunet/training/network_training/nnUNetTrainerV2.py", line 124, in initialize
    self.initialize_network()
  File "/home/smaijer/nnUNet/nnunet/training/network_training/nnUNetTrainerV2.py", line 167, in initialize_network
    self.network.cuda()
  File "/home/smaijer/.conda/envs/nn/lib/python3.9/site-packages/torch/nn/modules/module.py", line 688, in cuda
    return self._apply(lambda t: t.cuda(device))
  File "/home/smaijer/.conda/envs/nn/lib/python3.9/site-packages/torch/nn/modules/module.py", line 578, in _apply
    module._apply(fn)
  File "/home/smaijer/.conda/envs/nn/lib/python3.9/site-packages/torch/nn/modules/module.py", line 578, in _apply
    module._apply(fn)
  File "/home/smaijer/.conda/envs/nn/lib/python3.9/site-packages/torch/nn/modules/module.py", line 578, in _apply
    module._apply(fn)
  [Previous line repeated 989 more times]
  File "/home/smaijer/.conda/envs/nn/lib/python3.9/site-packages/torch/nn/modules/module.py", line 577, in _apply
    for module in self.children():
  File "/home/smaijer/.conda/envs/nn/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1618, in children
    for name, module in self.named_children():
  File "/home/smaijer/.conda/envs/nn/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1636, in named_children
    for name, module in self._modules.items():
RecursionError: maximum recursion depth exceeded while calling a Python object
